Node.js/boilerplate.js

```diff
@@ -156,7 +156,7 @@ class MQBoilerPlate {
       var queue = me.mqObj;
 
       // Describe how the Put should behave
-      pmo.Options = MQC.MQPMO_NO_SYNCPOINT;
+      pmo.Options = MQC.MQPMO_SYNCPOINT;
```

Comment:
Is boilerplate.js used in other samples? The transactional changes are for the request/response samples only. Is a global change to the behaviour the right approach or does boilerplate.js need to be extended to also support tx?